src/ui.button-toggle/UToggle.vue

```diff
@@ -58,20 +58,24 @@ const type = computed(() => {
   return props.multiple ? TYPE_CHECKBOX : TYPE_RADIO;
 });
 
-function updateSelectedValue(value, checked) {
+function updateSelectedValue(value: string | number, checked: boolean) {
   if (type.value === TYPE_RADIO) {
     selectedValue.value = value;
 
     return;
   }
 
   if (checked) {
-    const items = selectedValue.value || [];
+    const currentValue = selectedValue.value;
```

Comment:
Why do you need to put `selectedValue` in another variable? Can't you use `selectedValue.value` right away?